code/go/0chain.net/blobbercore/handler/grpc_helpers.go

```diff
@@ -36,7 +36,19 @@ func registerGRPCServices(r *mux.Router, server *grpc.Server) {
 			common.UserRateLimit(common.ToJSONResponse(WithConnection(UploadHandler)))(w, r)
 		})
 
-	_ = grpcGatewayHandler.HandlePath("POST", "/v1/file/download/{allocation}",
+	_ = grpcGatewayHandler.HandlePath("PUT", "/v1/file/upload/{allocation}",
```

Comment:
Why the error is ignored in each place?
```suggestion
	err = grpcGatewayHandler.HandlePath("PUT", "/v1/file/upload/{allocation}",
		if err != nil {
		log.Errorf("failed to add custom route: %v", err)
		panic(err)
	}
```

Reply:
okay sure